test/feature/detect_victor_test.exs

```diff
@@ -10,15 +10,15 @@ defmodule DetectVictorTest do
     assert Board.there_is_a_winner?("XX3OOO789", 3, "X", "O") == true
   end
 
-  test "detects player 1 victory" do
-    assert Board.winner("OXO4X67X9", "X", "O") == "X"
+  test "declares X is the winner" do
+    assert Board.winner("XXXOO6789", "X", "Y", "O", "U") == "Y"
```

Comment:
**thought:** Looking at these tests, do you think they're giving us any clues about our design here?